src/test/compile-fail/lint-unused-mut-variables.rs

```diff
@@ -8,6 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// revisions: lexical nll
+#![cfg_attr(nll, feature(nll))]
```

Comment:
We're getting [travis errors](https://travis-ci.org/rust-lang/rust/builds/350678719#L2394). The problem is that we have to update the `//~ ERROR` declarations in this file to account for the revisions.

For each line:

```
foo //~ ERROR bar
```

we have to change it to

```
foo //[lexical]~ ERROR bar
 //[nll]~^ ERROR bar
```